acceptance/Rakefile

```diff
@@ -0,0 +1,111 @@
+               'ssh'
+             end
+    node = node + roles.join(',') + '.'
+    n_new << node
+  end
+  nodes_final = n_new.join('-')
+  generate = "bundle exec beaker-hostgenerator"
+  generate += " #{nodes_final}"
+  generate += " > hosts.yaml"
+  sh generate
+  sh "cat hosts.yaml"
+end
+
+namespace :ci do
+  beaker_options = [
+    { name: '--hosts',
+      add_argument: {
+        name: 'hosts.yaml',
+        add_env: {
+          name: 'BEAKER_HOSTS_FILE',
+          message: 'Beaker hosts file'
+        }
+      } },
+    { name: '--tests',
+      add_argument: {
+        name: 'tests',
+        add_env: {
+          name: 'BEAKER_TESTS',
+          message: 'Beaker test path(s)'
+        }
+      } },
+    { name: '--preserve-hosts',
+      add_argument: {
+        name: 'never',
+        add_env: {
+          name: 'BEAKER_PREVERSE_HOSTS',
+          message: 'When should beaker keep hosts alive'
+        }
+      } }
+  ]
+  # The keyfile is separated so that it can be exclude if the value is not
+  # set. This is a rototiller bug QA-2761.
+  beaker_keyfile = {
+    name: '--keyfile',
+    add_argument: {
+      add_env: {
+        name: 'BEAKER_KEYFILE',
+        message: 'The private SSH key beaker will use to connect to SUTs'
+      }
+    }
+  }
+  namespace :test do
+    desc "Run bolt acceptance tests against a published gem"
+    task gem: :host_config
+    rototiller_task :gem do |task|
+      beaker_options << {
+        name: '--options',
+        add_argument: { name: 'config/gem/options.rb' }
+      }
+      task.add_env(name: 'GEM_VERSION', default: '> 0.1.0')
+      task.add_env(name: 'GEM_SOURCE',  default: 'https://rubygems.org')
+      task.add_env(name: 'SSH_USER',    default: 'root')
+      task.add_env(name: 'SSH_PASSWORD')
+      task.add_env(name: 'WINRM_USER', default: 'Administrator')
+      task.add_env(name: 'WINRM_PASSWORD')
+      task.add_command do |cmd|
+        cmd.name = 'bundle exec beaker'
+        cmd.add_option(*beaker_options)
+        cmd.add_option(beaker_keyfile) if ENV['BEAKER_KEYFILE']
+      end
+    end
+
+    desc "Run bolt acceptance tests against a git repo"
+    task git: :host_config
+    rototiller_task :git do |task|
+      beaker_options << {
+        name: '--options',
+        add_argument: { name: 'config/git/options.rb' }
+      }
+      task.add_env(name: 'GIT_SERVER', default: 'https://github.com')
+      task.add_env(name: 'GIT_FORK',   default: 'puppetlabs/bolt.git')
+      task.add_env(name: 'GIT_BRANCH', default: 'master')
+      task.add_env(name: 'GIT_SHA',    default: '')
+      task.add_env(name: 'SSH_USER',   default: 'root')
+      task.add_env(name: 'SSH_PASSWORD')
+      task.add_env(name: 'WINRM_USER', default: 'Administrator')
+      task.add_env(name: 'WINRM_PASSWORD')
+      task.add_command do |cmd|
+        cmd.name = 'bundle exec beaker'
+        cmd.add_option(*beaker_options)
+        cmd.add_option(beaker_keyfile) if ENV['BEAKER_KEYFILE']
```

Comment:
I copied `id_rsa-acceptance*` to the cwd, but beaker was unable to ssh to the host as root.

```
Spent 0.02 seconds tagging VMs
  Warning: Beaker does not support vmhostname to SSH to host, trying next available method.
  Warning: Skipping ip method to ssh to host as its value is not set. Refer to https://github.com/puppetlabs/beaker/tree/master/docs/how_to/ssh_connection_preference.md to remove this warning
  Warning: Beaker does not support hostname to SSH to host, trying next available method.
  Warning: Skipping ip method to ssh to host as its value is not set. Refer to https://github.com/puppetlabs/beaker/tree/master/docs/how_to/ssh_connection_preference.md to remove this warning
:paranoid is deprecated, please use :verify_host_key. Supported values are exactly the same, only the name of the option has changed.
  Warning: Try 1 -- Host lepf9q0hhyp5jri.delivery.puppetlabs.net unreachable: Net::SSH::AuthenticationFailed - Authentication failed for user root@lepf9q0hhyp5jri.delivery.puppetlabs.net
  Warning: Trying again in 3 seconds
:paranoid is deprecated, please use :verify_host_key. Supported values are exactly the same, only the name of the option has changed.
...
```

I deleted the pub/private keys, and ran with `BEAKER_KEYFILE=~/.ssh/id_rsa-acceptance`, which did work. Could you add `BEAKER_KEYFILE` to the list of environment variables that may need setting?